pcdsdevices/sample_delivery.py

```diff
@@ -0,0 +1,440 @@
+"""
+Module for the Sample Delivery System and related devices.
+"""
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal
+
+from .component import UnrelatedComponent as UCpt
+from .signal import PytmcSignal
+
+
+class ViciValve(Device):
+    """
+    A Vici Valve as used in the SDS Selector.
+
+    Parameters
+    ----------
+    prefix : str
+        The base PV for the Selector.
+
+    name : str
+        A name for the device.
+    """
+
+    req_pos = Cpt(PytmcSignal, ':ReqPos', io='io', kind='normal')
+    curr_pos = Cpt(PytmcSignal, ':CurrentPos', io='i', kind='normal')
+
+
+class Selector(Device):
+    """
+    A Selector for the sample delivery system.
+
```

Comment:
Should we be more clear that this is strictly an implementation of the M3 selector?

Reply:
Yeah, that'd probably be good